packages/environment-check/src/output/markdown.ts

```diff
@@ -3,12 +3,14 @@ import { Severity, UrlServiceType } from '../types';
 import type {
     Destination,
     DestinationResults,
-    Environment,
+    BASEnvironment,
     EnvironmentCheckResult,
     MarkdownWriter,
-    ResultMessage
+    ResultMessage,
+    VSCodeEnvironment
 } from '../types';
 import { t } from '../i18n';
+import { isAppStudio } from '@sap-ux/btp-utils';
```

Comment:
Ideally, the markdown writer doesn't have any knowledge about the environment. We should try to find a result structure that can be converted to markdown without knowing which environment we currently running in.

Reply:
See comment